src/agenda/index.js

```diff
@@ -253,7 +270,9 @@ export default class AgendaView extends Component {
     this.setScrollPadPosition(snapY, true);
 
     if (snapY === 0) {
-      this.enableCalendarScrolling();
+      this.enableCalendarScrolling(true);
+    } else {
+      this.enableCalendarScrolling(false);
     }
```

Comment:
Also here you can shorten the if statement to: `this.enableCalendarScrolling(snapY === 0);`

Reply:
Updated!